packages/dx-react-grid-bootstrap3/src/templates/table-detail-toggle-cell.test.jsx

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import { TableDetailToggleCell } from './table-detail-toggle-cell';
 
 const ENTER_KEY_CODE = 13;
 const SPACE_KEY_CODE = 32;
 
 describe('TableDetailToggleCell', () => {
+  it('should handle click with stopPropagation', () => {
+    const toggleExpanded = jest.fn();
+    const mockEvent = {
+      stopPropagation: jest.fn(),
+    };
+    const tree = shallow((
+      <TableDetailToggleCell
+        toggleExpanded={toggleExpanded}
+      />
+    ));
+
+    const clickHandlerButton = tree.find('td').prop('onClick');
```

Comment:
`clickHandlerButton` looks like an element, but in fact, it is a function. So, it can be renamed to `buttonClickHandler`.